src/Common/CodeEditor/CodeEditor.tsx

```diff
@@ -85,7 +87,7 @@ const CodeEditor: React.FC<CodeEditorInterface> & CodeEditorComposition = React.
         children,
         tabSize = 2,
         lineDecorationsWidth = 0,
-        height = 450,
+        height = DEFAULT_EDITOR_HEIGHT,
```

Comment:
Do we need to give height when adjustEditorHeightToContent is true?

Reply:
Changed the type such that we can only have one of the two properties at a time - either height or adjustEditorHeightToContent.

Also added a new minHeight prop